_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,114 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+---
+
+## Applicability
+
+This rule applies to any `img` element that is [visible][] except if the `src` [attribute value][] references an SVG document.
```

Comment:
It's because the user is capable of setting font family and size, color, and background of an SVG `text` element, which is what is required by the SC.
I'll document it in the background.

Reply:
👍
That does leave out cases where SVG shapes are used to "draw" letters without using `text`. But I guess this is rather unfrequent, usually needed, and anyway only passing the rule too often, so we're good.